ui/pages/SummaryData/reducers.js

```diff
@@ -43,21 +43,32 @@ export const loadSuccessStory = successStoryTypes => (dispatch) => {
 
 export const loadSavedVariants = ({ tag, gene = '' }) => (dispatch, getState) => {
   // Do not load if already loaded
+  let tags = [SHOW_ALL]
   if (tag) {
-    if (getState().savedVariantTags[tag]) {
+    const { savedVariantTags } = getState()
+    tags = tag.split(';').filter(t => !savedVariantTags[t])
+    if (tags.length === 0) {
       return
     }
   } else if (!gene) {
     return
   }
 
   dispatch({ type: REQUEST_SAVED_VARIANTS })
-  new HttpRequestHelper(`/api/summary_data/saved_variants/${tag}`,
+  new HttpRequestHelper(`/api/summary_data/saved_variants/${tags.join(';')}`,
     (responseJson) => {
       if (tag && !gene) {
+        if (tags[0] === SHOW_ALL) {
```

Comment:
this case is impossible - you can't return all variants with no gene filter applied